troubleshooting/troubleshooting-vse-path-too-long-exception.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Path Too Long Exception when Building Project in Microsoft Visual Studio
+description: AppBuilder Projects with fully qualified file names longer that 260 symbols fail to build in Visual Studio Extension.
+type: troubleshooting
+page_title: VS Extension Build Fails when File Path Length is Longer than 260 Symbols 
+slug: vse-path-too-long-exception
+position:
+tags: appbuilder, vs-extension, build
+teampulseid:
+ticketid:  1044168, 1001926
+pitsid:
+
```

Comment:
* Remove the empty line here
* Is this empty line something that comes from the template?